src/features/search/dropdown/search-results-card.tsx

```diff
@@ -45,11 +53,13 @@ const CardActions: React.FC<CardActionsProps> = ({ isLoading, hasError, hasResul
   );
 };
 
-export const SearchResultsCard: React.FC = () => {
-  const {
-    query: { data, isLoading, error },
-    searchTerm,
-  } = useSearch();
+export function SearchResultsCard({
+  searchResponse,
+}: {
+  searchResponse: ReturnType<typeof useSearchQuery>;
+}) {
+  const { data, isLoading, error } = searchResponse;
+  const searchTerm = useAppSelector(selectSearchTerm);
```

Comment:
does the search term need to be in global state?

Reply:
We had this redux search slice for a long time (handling search term and other search-related state) when we had search components scattered in multiple places. We should be able to consolidate this and eliminate the slice.